src/RayServiceProvider.php

```diff
@@ -68,6 +68,7 @@ protected function registerSettings(): self
                 'send_queries_to_ray' => env('SEND_QUERIES_TO_RAY', false),
                 'send_requests_to_ray' => env('SEND_REQUESTS_TO_RAY', false),
                 'send_views_to_ray' => env('SEND_VIEWS_TO_RAY', false),
+                'send_exceptions_to_ray' => env('SEND_EXCEPTIONS_TO_RAY', false),
```

Comment:
this doesn't hold the same default vs config file. Intentional?

Reply:
Right, the exceptions should be sent by default. I'll fix. Thank for the heads up @crynobone